src/components/Landing/QAs.tsx

```diff
@@ -130,5 +218,48 @@ export default function QAs() {
         alt: t`It'd be a lot cooler if you did`,
       },
     },
+    {
+      q: t`Who is Peel?`,
+      a: [
+        <Trans>
+          <a
+            href="https://juicebox.money/#/p/peel"
```

Comment:
sick...

Technically this could be a react-router `<Link />` becuase it's an internal link

`<Link to="/p/peel">Peel</Link>`